### ISSUE TITLE
[Rule change] add ingnore param to no-equal-arguments

### ISSUE THREAD

**Issue by grafovdenis** [dart-code-checker/dart-code-metrics#339]
**What rule do you want to change?**

[No equal arguments](https://github.com/dart-code-checker/dart-code-metrics/blob/master/doc/rules/no-equal-arguments.md)

**How will the change be implemented? (New option, new default behavior, etc.)?**

Make arguments of this rule configurable.

**Example:**

```yml
dart_code_metrics:
  ...
  rules:
    ...
    - no-equal-arguments:
        ignore: [height, width]
```

**Please provide some example code that this change will affect:**

For example, i'd like to create symmetrical geometrical figure (square, circle) using Flutter.

```dart

const size = 20.0;

final circle = Container(
      width: size,
      height: size,  // LINT
      decoration: BoxDecoration(
        shape: BoxShape.circle,
        color: Colors.red,
      ),
    );

final square = Container(
      width: size,
      height: size,  // LINT
      color: Colors.red,
    );

```

**What does the rule currently do for this code?**

Lints height parameter since it's an equal argument.

**What will the rule do after it's changed?**

Allow certain arguments to ignore this rule.

**Comment by incendial** [comment]
@grafovdenis thats actually a great idea. There is another issue #327 and I initially suggested to add an ignore comment, but your idea is way more better.

Will support this soon. I'm not sure about the exact parameter name, though.

**Comment by incendial** [comment]
3.3.0 version with the improvement is now available on pub.
